### ISSUE TITLE
Illegal instruction

### ISSUE THREAD

**Issue by dm-coding** [adrienbrault/docker-plex-arm#1]
Well this is very strange. I too am using it on a Scaleway C1 but I encounter the following error:

$ docker logs plex
Illegal instruction (core dumped)
6 3000 /config/Library/Application Support
8192
Illegal instruction (core dumped)
<etc>

Command:

$ docker run -d \
    --name plex \
    --restart always \
    -p 32400:32400 \
    -h host.fqdn \
    -v /root/plex-config:/config \
    -v /root \
    adrienbrault/armhf-plex

**Comment by adrienbrault** [comment]
@djcf Yes, I rebuilt the image a few days ago and pushed it to docker hub, but encountered the same error.

I rolled back to the one I had previously built, but forgot to update docker hub. That is now done, please try to pull again, here's what I use right now `8a351ad16813`:

```
root@mille-feuilles:~# docker images
REPOSITORY                  TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
adrienbrault/armhf-plex     latest              8a351ad16813        8 weeks ago         415.6 MB
```
